platform-sdk/swirlds-config-processor/build.gradle.kts

```diff
@@ -26,7 +26,7 @@ testModuleInfo { requires("org.junit.jupiter.api") }
 
 tasks.register<AntlrTask>("generateParser") {
     conventionMapping.map("antlrClasspath") {
-        configurations.detachedConfiguration(dependencies.create("org.antlr:antlr4:4.13.1"))
+        configurations.detachedConfiguration(dependencies.create("org.antlr:antlr4:4.11.1"))
```

Comment:
is there any particular reason for the downgrade?

Reply:
We use 4.11.1 at some other places. By doing the downgrade it is always the same version now.